### ISSUE TITLE
A question about LES modeling of turbinesFoam

### ISSUE THREAD

**Issue by GetChaos** [hcOnel/NREL5MW-turbinesFoam#3]
Hello bro,
I saw one of your issues about the LES modeling in turbinesFoam here: https://github.com/turbinesFoam/turbinesFoam/issues/281. I came across the same problem. The velocity magnitude does not recovery in LES modeling. How did you handle the problem at last? Do you have some advices? Thanks so much!

**Comment by hcOnel** [comment]
Hi,
My comments on wake not recovering in an LES are:
1. Absence of inflow turbulence (such as methods like DFSEM),
2. Low grid resolution in the wake region
3. Improper SGS modeling
The paper recommended by @petebachant discusses the SGS modeling part.

**Comment by GetChaos** [comment]
Hi hcOnel,
Thanks for your quick response. 
May I ask,  do I have to add the inflow turbulence to model the wake recovering in an LES? I mean in RANS modeling, I didn't apply inflow turbulence but the wake velocity can still recovery.  That really confused me. I'll try to improve the grid resolution.
Thanks again for your comments!

**Comment by hcOnel** [comment]
Yes, with RANS, you can set the inflow turbulence by adjusting the field variables (i.e. k, epsilon...) at the inlet boundary. With LES however, you have to use some additional model at the inlet plane.
Please see:
https://www.researchgate.net/publication/256089177_A_New_Divergence_Free_Synthetic_Eddy_Method_for_the_Reproduction_of_Inlet_Flow_Conditions_for_LES

**Comment by GetChaos** [comment]
Thanks so much!
